<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>280</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>280</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabSavingAndLoading">
      <attribute name="title">
       <string>Saving and Loading</string>
      </attribute>
      <widget class="QCheckBox" name="reloadTilesetImages">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>377</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Reload tileset images when they change</string>
       </property>
      </widget>
      <widget class="QComboBox" name="layerDataCombo">
       <property name="geometry">
        <rect>
         <x>161</x>
         <y>10</y>
         <width>220</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>XML</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Base64 (uncompressed)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Base64 (gzip compressed)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Base64 (zlib compressed)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CSV</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>12</y>
         <width>144</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Store tile layer data as:</string>
       </property>
       <property name="buddy">
        <cstring>layerDataCombo</cstring>
       </property>
      </widget>
      <widget class="QCheckBox" name="enableDtd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>39</y>
         <width>377</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Not enabled by default since a reference to an external DTD is known to cause problems with some XML parsers.</string>
       </property>
       <property name="text">
        <string>Include &amp;DTD reference in saved maps</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabInterface">
      <attribute name="title">
       <string>Interface</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>379</width>
         <height>79</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Language:</string>
          </property>
          <property name="buddy">
           <cstring>languageCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="languageCombo">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="openGL">
          <property name="text">
           <string>Hardware &amp;accelerated drawing (OpenGL)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabBackgroundAndGrid">
      <attribute name="title">
       <string>Background and Grid</string>
      </attribute>
      <widget class="Tiled::Internal::ColorButton" name="backgroundColor">
       <property name="geometry">
        <rect>
         <x>134</x>
         <y>10</y>
         <width>27</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>116</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Background color:</string>
       </property>
      </widget>
      <widget class="QToolButton" name="addGridStyle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>27</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="tiled.qrc">
         <normaloff>:/images/16x16/style-add.png</normaloff>:/images/16x16/style-add.png</iconset>
       </property>
      </widget>
      <widget class="QListView" name="gridStylesView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>471</width>
         <height>161</height>
        </rect>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="removeGridStyle">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>27</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="tiled.qrc">
         <normaloff>:/images/16x16/style-delete.png</normaloff>:/images/16x16/style-delete.png</iconset>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tiled.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
